mule-extensions-api/src/main/java/org/mule/runtime/extension/api/stereotype/ChainStereotype.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.stereotype;
+
+/**
+ * {@link StereotypeDefinition} for {@code body}/chain of components defined through a DSL.
```

Comment:
it's not a body or chain.. it as chain. It's important for this to be prescriptive